src/style.scss

```diff
@@ -172,4 +172,14 @@
     font-size: var(--auro-text-body-size-sm);
     line-height: var(--auro-text-body-height-default);
   }
+}
+
+:host([md]) {
+  .pane {
+    font-size: var(--auro-text-body-size-default);
+    grid-template-rows: 1.625rem, 1.25rem;
+    padding-top: var(--auro-size-md);
+    height: 5.625rem;
+    min-width: 4.25rem;
+  }
```

Comment:
The scope of this update is to have an attribute that locks the UI into the same shape as the mobile-first UI. 

This code is not exactly that. 

I would assume that we could use the CSS from the mobile-first UI and put that into a mixin and reuse the exact same code under the new selector. 

That way we aren't duplicating code and have to maintain the same UI in two places.